spec/models/user_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
+describe User, type: :model do
+  let(:username) { 'Factorial' }
+  let(:user) { FactoryGirl.create(:user, username: username) }
```

Comment:
RE: Why add a username variable?
The factorygirl was sequencing so checking the username would break the test occasionally. This way, we specify a username and can check to see if it stays the same.